Src/DFFWizard.pas

```diff
@@ -46,26 +46,24 @@ implementation
   Vcl.Controls, Vcl.Menus;
 
 { TDFFWizard }
-
-  //TODO: Should register the frame, so it can be saved in the layout
   //TODO: Should add a menu item, with a settings wheel, so people can change the hotkey
 
 procedure TDFFWizard.BindKeyboard(
   const BindingServices: IOTAKeyBindingServices);
 begin
-  BindingServices.AddKeyBinding([ShortCut(Ord('T'), [ssCtrl])], ShowForm, nil, 0, '', 'Nogen file finder');
+  BindingServices.AddKeyBinding([TextToShortCut('Ctrl+T')], ShowForm, nil, kfImplicitShift, '', '');
 end;
 
 constructor TDFFWizard.Create;
 begin
-  FFilesIndexingThread := TFilesIndexingThread.Create(DoNewFilesIndexed);
-  FFilesIndexingThread.Start;
+//  FFilesIndexingThread := TFilesIndexingThread.Create(DoNewFilesIndexed);
+//  FFilesIndexingThread.Start;
 end;
 
 procedure TDFFWizard.DoNewFilesIndexed(aFiles: TList<string>);
 begin
-  if Assigned(FForm) then
-    FForm.Frame.SetFiles(aFiles);
+//  if Assigned(FForm) then
+//    FForm.Frame.SetFiles(aFiles);
 end;

```

Comment:
Do you still need this procedure? And if so, since there is no procedure body, can it just be shortened?
```pas
procedure TDFFWizard.DoNewFilesIndexed(aFiles: TList<string>);
````